operator/src/main/java/org/keycloak/operator/controllers/KeycloakRealmImportJobDependentResource.java

```diff
@@ -147,7 +145,7 @@ private void buildKeycloakJobContainer(Container keycloakContainer, KeycloakReal
 
         var override = "--override=false";
 
-        var runBuild = !keycloakContainer.getArgs().contains(KeycloakDeploymentDependentResource.OPTIMIZED_ARG) ? "/opt/keycloak/bin/kc.sh --verbose build && " : "";
```

Comment:
I'm probably doing something wrong but even if I do something like this with the current main:
```
kc.sh build --db=postgres
kc.sh start --http-enabled=true --hostname-strict=false --optimized
```
I get an error. So not sure why it worked like that in the Operator before.

Reply:
Do you mean if you then do an import with the env setting for health-enabled after that it errors?

Reply:
Never mind, I think I just got confused here. 🙈

However, I do still wonder what is the benefit of disabling health in import. It doesn't help perf too much.